src/app/tissue/tissue.component.ts

```diff
@@ -158,7 +158,10 @@ export class TissueComponent implements OnInit {
               if ( jsonData instanceof Array ) {
                 jsonData.forEach((val) => {
                   let nameValue = NameValue.parse(val);
-                  this.oncHistoryDetail[nameValue.name] = nameValue.value;
+                  if (nameValue.name && nameValue.name.indexOf( '.' ) != -1) {
+                    nameValue.name = nameValue.name.substr( nameValue.name.indexOf( "." ) + 1);
```

Comment:
I personally prefer .split over .substr. How does the team feel? @gmakharat @Anzori-Ghurtchumelia @pegahtah